components/build/tekton-chains/chains-certs.yaml

```diff
@@ -59,7 +59,9 @@ spec:
             - -c
             - |
               echo "Waiting for secret and configmaps to be present"
-              until oc get secret chains-ca-cert -n tekton-chains
+              until oc get secret chains-ca-cert -n tekton-chains &&\
```

Comment:
Would `oc wait` be a bit more helpful here? The loop wouldn't be required.

Reply:
Doesn't look that this would  work here as `oc wait` requires a status field.

Reply:
Good call.